src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1770 @@
+        "prob_cure_stage2a": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2a cervical cancer",
+        ),
+        "prob_cure_stage2b": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2b cervical cancer",
+        ),
+        "prob_cure_stage3": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 3 cervical cancer",
+        ),
+        "r_death_cervical_cancer": Parameter(
+            Types.REAL,
+            "probability per month of death from cervical cancer amongst people with stage 4 cervical cancer",
+        ),
+        "r_vaginal_bleeding_cc_stage1": Parameter(
+            Types.REAL, "rate of vaginal bleeding if have stage 1 cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage2a": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 2a cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage2b": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 2b cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage3": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 3 cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage4": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 4 cervical cancer"
+        ),
+        "prob_referral_biopsy_given_vaginal_bleeding": Parameter(
+            Types.REAL, "probability of being referred for a biopsy if presenting with vaginal bleeding"
+        ),
+        "sensitivity_of_biopsy_for_cervical_cancer": Parameter(
+            Types.REAL, "sensitivity of biopsy for diagnosis of cervical cancer"
+        ),
+        "sensitivity_of_xpert_for_hpv_cin_cc": Parameter(
+            Types.REAL, "sensitivity of xpert for presence of hpv, cin or cervical cancer"
+        ),
+        "sensitivity_of_via_for_cin_cc": Parameter(
+            Types.REAL, "sensitivity of via for cin and cervical cancer bu stage"
+        ),
+        "prob_xpert_screen": Parameter(
+            Types.REAL, "probability of xpert screening"
+        ),
+        "prob_via_screen": Parameter(
+            Types.REAL, "probability of via screening"
+        ),
+        "prob_thermoabl_successful": Parameter(
+            Types.REAL, "probability of thermoablation treatment successful in removing CIN (ce_hpv_cc_status set to none)"
+        ),
+        "prob_cryotherapy_successful": Parameter(
+            Types.REAL, "probability of cryotherapy treatment successful in removing CIN (ce_hpv_cc_status set to none)"
+        ),
+        "transition_testing_year": Parameter(
+            Types.REAL, "year testing recommendation switches from VIA to Xpert"
+        ),
+        "transition_screening_year": Parameter(
+            Types.REAL, "year screening recommendation switches from Cryo to Thermo"
+        ),
+        "min_age_hpv": Parameter(
```

Comment:
reminder to check all parameters are actually being used - several seem not to be.